src/parseSerialize/schemas.ts

```diff
@@ -329,15 +329,11 @@ export const setScriptSchemaV1: TSchema = {
   ],
 }
 
-// This schema is not 100% correct for transfer-v1, it's a workaround for an issue with the Ledger app in the wallet ui
-// Once ledger app is fixed, this will no longer be needed
-// see the issue: https://github.com/iicc1/ledger-app-lto/issues/3
-export const transferSchemaV1: TSchema = {
+export const transferSchemaV2: TSchema = {
   type: 'object',
   schema: [
```

Comment:
There should be attachment length field before the attachment:
https://docs.ltonetwork.com/v/edge/protocol/public/transactions/transfer-transaction

Reply:
That is added on the serializer for every array field :)

https://github.com/ltonetwork/lto-transactions/blob/1517c5916cccbed8857165733166c002f1bcf466/src/parseSerialize/serialize.ts#L23